app/controllers/api/v2/template_controller.rb

```diff
@@ -1,52 +1,33 @@
 module Api
   module V2
     class TemplateController < ::Api::V2::BaseController
+      include ::Foreman::Controller::Parameters::TemplateParams
+
+      def_param_group :template_params do
```

Comment:
this could conflict with param group from core, perhaps change the name to `foreman_templates_sync_params` or similar?